specification/app/resource-manager/Microsoft.App/preview/2023-11-02-preview/examples/AppSpaceResources_CreateOrUpdate.json

```diff
@@ -0,0 +1,80 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "rg",
+    "appSpaceName": "testappSpace0",
+    "spaceResourceName": "testappSpaceResource0",
+    "api-version": "2023-11-02-preview",
+    "spaceResourceEnvelope": {
+      "location": "East US",
+      "properties": {
+        "resourceId": "/subscriptions/34adfa4f-cedf-4dc0-ba29-b6d1a69ab345/resourceGroups/rg/providers/Microsoft.Web/sites/testAppSpaceWebApp",
+        "friendlyName": "Test Resource 0 - Web App",
+        "resourceLocation": "West US",
+        "repo": {
+          "url": "github.com/myrepo",
+          "branch": "master"
+        },
+        "deployment": {
+          "id": "/subscriptions/34adfa4f-cedf-4dc0-ba29-b6d1a69ab345/resourceGroups/rg/providers/Microsoft.Resources/deployments/testAppSpaceWebAppDeployment",
+          "provisioningState": "Succeeded"
```

Comment:
can you please exaplin a bit about this parameter ?

Why do you need a reference to the deployment resource for the parent resource deployment ?

Reply:
How does your service use this information ?

Reply:
The portal UX folks will use this information to check on deployments from their end - they want us to store the information that then they can use for lookups rather than us doing any sort of checks from the backend. 

Basically an app space resource is a way to track a resource in another RP and have some information about it so that portal can display it in a different way/grouping/send reqs to other RPs to get info.